### PR TITLE
Changed Footer and ThemeToggle to accommodate new designs

### PR DESCRIPTION
**Pull request by ShimiSun**
#553

### CODE REVIEW

**Comment by ShimiSun** [comment]
> Looks good to me. Only comment is did Chris want the color for dark mode to be `blue!` still or something else?

Since we don't have the `dark` mode designs yet, I kept it as is until further notice.

**Comment by britt6612** [comment]
One thing I noticed was the `border-radius` is `4px `is designs and `12px` in preview. Unless these were changed then ignore my comment

**Comment by ShimiSun** [comment]
> One thing I noticed was the `border-radius` is `4px `is designs and `12px` in preview. Unless these were changed then ignore my comment

Good catch! Fixed

File: aries-site/src/components/content/ThemeModeToggle.js [hunk 1]
```diff
@@ -8,23 +8,22 @@ export const ThemeModeToggle = () => {
...
-          pad={{ vertical: 'small', horizontal: 'medium' }}
-          responsive={false}
-          round="xlarge"
-          width="small"
-        >
-          {icon}
-          {label}
-        </Box>
-      </Button>
+    <Box
```
Comment (halocline) [inline aries-site/src/components/content/ThemeModeToggle.js]: Curious as to the rationale for eliminating the `Button`?

**Comment by britt6612** [comment]
Screen Shot 2020-03-25 at 5 26 28 PM
Screen Shot 2020-03-25 at 5 26 22 PM

I did also notice the width and height were bigger then in designs. I changed to what was in the design so `width: 139px` and `height: 40px`  In the second screen shot it looks closer to designs but Im not sure if there was a reason for leaving this bigger

File: aries-site/src/components/content/ThemeModeToggle.js (hunk 1 above)
Comment (ShimiSun) [reply aries-site/src/components/content/ThemeModeToggle.js]: Good question, the focus was off due to the Button, it didn't adjust to the content of internal Box, so I moved the onClick inside the Box so the interactive element will be the Box and the focus will be accurate to the Box content, and then the Button didn't have any props and was just a redundant element in the DOM - so I removed the Button. 
We can take another approach of replacing the old Box rendering-approach with a Button, but I continued in the direction that was existing and usage of a single component. 
I'm willing to explore the latter.

**Comment by ShimiSun** [comment]
> ...Im not sure if there was a reason for leaving this bigger

The reason is to have a constant t-shirt size value so the button size won't flip sizes between dark and light modes.

**Comment by britt6612** [comment]
> > ...Im not sure if there was a reason for leaving this bigger
> 
> The reason is to have a constant t-shirt size value so the button size won't flip sizes between dark and light modes.

Makes sense! Everything else LGTM

File: aries-site/src/components/content/ThemeModeToggle.js (hunk 1 above)
Comment (ShimiSun) [reply aries-site/src/components/content/ThemeModeToggle.js]: I was intrigued by this question and refactored the Box to be a Button wrapped with a Box, let me know what you think.

**Comment by ShimiSun** [comment]
@britt6612 thanks for the feedback, I made another commit to refactor Box to Button, and I think the spacing looks better now, at least on Desktop screen.

**Comment by britt6612** [comment]
> @britt6612 thanks for the feedback, I made another commit to refactor Box to Button, and I think the spacing looks better now, at least on Desktop screen.

welcome I'll check it out with the new change!

**Comment by britt6612** [comment]
@ShimiSun looks great with the refactor to `button`

**Pull request merged**